vscode/src/commands/execute/explain.ts

```diff
@@ -50,7 +50,7 @@ export async function explainCommand(
 
     return {
         text: prompt,
-        submitType: 'user-newchat',
+        submitType: 'user',
```

Comment:
```suggestion
        submitType: 'user-newchat',
```

Do we want to include changes unrelated to the terminal output command?